ansible/adhoc/lock_unlock_instances.yml

```diff
@@ -0,0 +1,11 @@
+---
+
+- hosts: "{{ target_hosts | default('cluster') }}"
```

Comment:
Now I understand why this is required, I don't really like the name `target_hosts`, I kinda feel it should be "namespaced" via a prefix to this lock/unlock business.

Maybe:
`lock_unlock_hosts` (or `_group`) and `lock_unlock_action` would be better?

But I could live with it 🤷 .